### DISCUSSION TITLE
Run failed: on-push - main

### DISCUSSION THREAD

**Discussion by MJDelas** [greenelab/lab-website-template#286] [Q&A] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://github.com/MJDelas/delas-lab-website

### Version of Lab Website Template you are using

1.1.6

### Description

I updated some text on one of the `index.md` files but I am getting the following error after pushing. It seems related to ubuntu/ruby rather than the template but any help would be greatly appreciated. 

```
[build-site / build-site](https://github.com/MJDelas/delas-lab-website/actions/runs/11323529466/job/31486417261#step:4:8)
The current runner (ubuntu-24.04-x64) was detected as self-hosted because the platform does not match a GitHub-hosted runner image (or that image is deprecated and no longer supported).
In such a case, you should install Ruby in the $RUNNER_TOOL_CACHE yourself, for example using https://github.com/rbenv/ruby-build
You can take inspiration from this workflow for more details: https://github.com/ruby/ruby-builder/blob/master/.github/workflows/build.yml
$ ruby-build 3.1.4 /opt/hostedtoolcache/Ruby/3.1.4/x64
Once that completes successfully, mark it as complete with:
$ touch /opt/hostedtoolcache/Ruby/3.1.4/x64.complete
It is your responsibility to ensure installing Ruby like that is not done in parallel.
```

**Comment by vincerubinetti** [comment, marked as answer]
A bunch of people have reported this in the last few days. Please follow this issue for updates:

https://github.com/greenelab/lab-website-template/issues/285

**Reply by MJDelas** [reply]
Yes, thanks so much! Since this morning I found the issue and I'm following.
